### DISCUSSION TITLE
Check scheduled status

### DISCUSSION THREAD

**Discussion by rmahoney10** [jdholtz/auto-southwest-check-in#187] [Q&A] [answered]
I've used this program successfully a few times and it's worked great!

Is there a way to see which flights are currently scheduled for check in? I know I can just run the schedule command again, but it'd be nice to be able to see what's already scheduled for check in and at what time. I'm also interested in make sure the program is still running before/after device sleep. Any advice?

**Comment by jdholtz** [comment, marked as answer]
> Is there a way to see which flights are currently scheduled for check in?

You can currently see it with the notifications you received (if they are set up) or by looking at the script's output since you started it. Would you want a way to do it on-demand? Something like hitting 'Enter' in the prompt to display the currently scheduled flights?

> I'm also interested in make sure the program is still running before/after device sleep. Any advice?

There is an FAQ question on that in the [Readme](https://github.com/jdholtz/auto-southwest-check-in#faq). Do you want any further clarification?

**Reply by sidneyvanness** [reply]
I agree something like this would be useful.

**Reply by jdholtz** [reply]
@sidneyvanness could you be a little more specific on what exactly would be useful? Is what the script does currently not sufficient?
